databases/snowflake/batch_prediction_api/snow_batch_prediction_api_query_s3.ipynb

```diff
@@ -0,0 +1,512 @@
+     "text": [
+      "completed INITIALIZING\n",
+      "completed RUNNING\n",
+      "status is now COMPLETED\n"
+     ]
+    }
+   ],
+   "source": [
+    "if response.status_code == 202:\n",
+    "\n",
+    "    while job['status'] == 'INITIALIZING':\n",
+    "        time.sleep(1)\n",
+    "        response = session.get(job['links']['self'])\n",
+    "        response.raise_for_status()\n",
+    "        job = response.json()\n",
+    "        \n",
+    "    print('completed INITIALIZING')\n",
+    "        \n",
+    "    if job['status'] == 'RUNNING':\n",
+    "\n",
+    "        while job['status'] == 'RUNNING':\n",
+    "            time.sleep(1)\n",
+    "            response = session.get(job['links']['self'])\n",
+    "            response.raise_for_status()\n",
+    "            job = response.json()\n",
+    "            \n",
+    "    print('completed RUNNING')\n",
+    "    print('status is now {status}'.format(status=job['status']))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-22T08:17:10.883806Z",
+     "start_time": "2020-03-22T08:17:10.880532Z"
+    }
+   },
+   "source": [
+    "### Truncate and Reload STG Staging Table with Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-22T08:20:16.576208Z",
+     "start_time": "2020-03-22T08:20:13.950591Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<snowflake.connector.cursor.SnowflakeCursor at 0x1233d0f10>,\n",
+       " <snowflake.connector.cursor.SnowflakeCursor at 0x1231244d0>]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# multi-statement executions\n",
+    "# https://docs.snowflake.com/en/user-guide/python-connector-api.html#execute_string\n",
+    "\n",
+    "# truncate and load STG schema table with scored results\n",
+    "sql = \"truncate titanic.stg.PASSENGERS_SCORED_BATCH_API; \" \\\n",
+    "    \" copy into titanic.stg.PASSENGERS_SCORED_BATCH_API from @S3_SUPPORT/titanic/community/\" + JOB_NAME + \"_scored.csv\" \\\n",
+    "    \" FILE_FORMAT = 'DEFAULT_CSV' ON_ERROR = 'ABORT_STATEMENT' PURGE = FALSE;\"\n",
+    "ctx.execute_string(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-22T08:17:34.107279Z",
+     "start_time": "2020-03-22T08:17:34.104603Z"
+    }
+   },
+   "source": [
+    "### Update Presentation Target Table With Results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-03-22T08:20:24.458764Z",
+     "start_time": "2020-03-22T08:20:16.578812Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<snowflake.connector.cursor.SnowflakeCursor at 0x12326dd90>,\n",
```

Comment:
Not sure what these SnowflakeCursors are?

Reply:
i assume some local hex memory address or something.